Application/EdFi.Ods.Api.IntegrationTestHarness/appsettings.json

```diff
@@ -14,6 +14,7 @@
     "EncryptSecrets": false,
     "UseReverseProxyHeaders": false,
     "BearerTokenTimeoutMinutes": "30",
+	"BearerTokenPerClientLimit": "-1",
```

Comment:
This setting turns off the maximum token limit in the integration test harness to accommodate clients such as the SmokeTest utility and Postman integration test suite, which generate numerous tokens during regular operation.